### PR TITLE
Fix incorrect log information and log structure

### PR DESCRIPTION
**Pull request by yangjunmyfm192085**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Fix incorrect log information
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
Fix incorrect log information
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by yangjunmyfm192085 — COMMENTED** [review]
/cc @aojea

File: pkg/util/netsh/netsh.go [hunk 1]
```diff
@@ -147,7 +147,7 @@ func (runner *runner) EnsureIPAddress(args []string, ip net.IP) (bool, error) {
 
 // DeleteIPAddress checks if the specified IP address is present and, if so, deletes it.
 func (runner *runner) DeleteIPAddress(args []string) error {
-	klog.V(4).InfoS("Running netsh interface IPv4 delete address", "IP", args)
+	klog.V(4).InfoS("Running netsh interface IPv4 delete address", "args", args)
```
[the remark this answers is not in the document]
Comment (yangjunmyfm192085) [reply pkg/util/netsh/netsh.go]: @danwinship thanks for review, I understand that it's really not worth it to just fix this error.
When I checked the proxy code, I found that there is still a file without log structure, I try to update together, could you please take a look again, thanks.

**Review by yangjunmyfm192085 — COMMENTED** [review]
/retest

File: pkg/util/iptables/iptables.go [hunk 2]
```diff
@@ -412,15 +412,15 @@ func (runner *runner) restoreInternal(args []string, data []byte, flush FlushFla
...
+				klog.ErrorS(err, "Failed to close iptables locks")
 			}
 		}(locker)
 	}
 
 	// run the command and return the output or an error including the output and error
 	fullArgs := append(runner.restoreWaitFlag, args...)
 	iptablesRestoreCmd := iptablesRestoreCommand(runner.protocol)
-	klog.V(4).Infof("running %s %v", iptablesRestoreCmd, fullArgs)
+	klog.V(4).InfoS("Running", "iptablesRestoreCmd", iptablesRestoreCmd, "arguments", fullArgs)
```
Comment (danwinship) [inline pkg/util/iptables/iptables.go:423]:
```suggestion
	klog.V(4).InfoS("Running", "command", iptablesRestoreCmd, "arguments", fullArgs)
```

File: pkg/util/iptables/iptables.go [hunk 3]
```diff
@@ -464,7 +464,7 @@ func (runner *runner) runContext(ctx context.Context, op operation, args []strin
 	iptablesCmd := iptablesCommand(runner.protocol)
 	fullArgs := append(runner.waitFlag, string(op))
 	fullArgs = append(fullArgs, args...)
-	klog.V(5).Infof("running iptables: %s %v", iptablesCmd, fullArgs)
+	klog.V(5).InfoS("Running iptables", "iptablesCmd", iptablesCmd, "arguments", fullArgs)
```
Comment (danwinship) [inline pkg/util/iptables/iptables.go:467]:
```suggestion
	klog.V(5).InfoS("Running", "command", iptablesCmd, "arguments", fullArgs)
```

for consistency

File: pkg/util/iptables/iptables.go [hunk 4]
```diff
@@ -492,7 +492,7 @@ func trimhex(s string) string {
 // of hack and half-measures.  We should nix this ASAP.
 func (runner *runner) checkRuleWithoutCheck(table Table, chain Chain, args ...string) (bool, error) {
 	iptablesSaveCmd := iptablesSaveCommand(runner.protocol)
-	klog.V(1).Infof("running %s -t %s", iptablesSaveCmd, string(table))
+	klog.V(1).InfoS("Running iptables", "iptablesSaveCmd", iptablesSaveCmd, "table", string(table))
```
Comment (danwinship) [inline pkg/util/iptables/iptables.go:495]:
```suggestion
	klog.V(1).InfoS("Running", "command", iptablesSaveCmd, "table", string(table))
```

File: pkg/util/iptables/iptables.go [hunk 5]
```diff
@@ -572,7 +572,7 @@ func (runner *runner) Monitor(canary Chain, tables []Table, reloadFunc func(), i
 		_ = utilwait.PollImmediateUntil(interval, func() (bool, error) {
 			for _, table := range tables {
 				if _, err := runner.EnsureChain(table, canary); err != nil {
-					klog.Warningf("Could not set up iptables canary %s/%s: %v", string(table), string(canary), err)
+					klog.InfoS("Could not set up iptables canary", "table", string(table), "canary", string(canary), "err", err)
```
Comment (danwinship) [inline pkg/util/iptables/iptables.go:575]: hm, those casts were never needed

```suggestion
					klog.ErrorS(err, "Could not set up iptables canary", "table", table, "chain", canary)
```

File: pkg/util/iptables/iptables.go [hunk 6]
```diff
@@ -584,11 +584,10 @@ func (runner *runner) Monitor(canary Chain, tables []Table, reloadFunc func(), i
 			if exists, err := runner.ChainExists(tables[0], canary); exists {
 				return false, nil
 			} else if isResourceError(err) {
-				klog.Warningf("Could not check for iptables canary %s/%s: %v", string(tables[0]), string(canary), err)
+				klog.InfoS("Could not check for iptables canary", "table", string(tables[0]), "canary", string(canary), "err", err)
```
Comment (danwinship) [inline pkg/util/iptables/iptables.go:587]:
```suggestion
				klog.ErrorS(err, "Could not check for iptables canary", "table", tables[0], "chain", canary)
```

File: pkg/util/iptables/iptables.go [hunk 7]
```diff
@@ -584,11 +584,10 @@ func (runner *runner) Monitor(canary Chain, tables []Table, reloadFunc func(), i
...
 				return false, nil
 			}
-			klog.V(2).Infof("iptables canary %s/%s deleted", string(tables[0]), string(canary))
-
+			klog.V(2).InfoS("Iptables canary deleted", "table", string(tables[0]), "canary", string(canary))
```
Comment (danwinship) [inline pkg/util/iptables/iptables.go:590]:
```suggestion
			klog.V(2).InfoS("IPTables canary deleted", "table", tables[0], "chain", canary)
```

**Review by danwinship — COMMENTED** [review]
This still needs review from someone more familiar with klog stuff

**Review by yangjunmyfm192085 — COMMENTED** [review]
/retest

**Review by yangjunmyfm192085 — COMMENTED** [review]
/cc @yuzhiquan

**Review by yangjunmyfm192085 — COMMENTED** [review]
/retest

File: pkg/util/iptables/iptables.go [hunk 8]
```diff
@@ -355,7 +355,7 @@ func (runner *runner) SaveInto(table Table, buffer *bytes.Buffer) error {
 	// run and return
 	iptablesSaveCmd := iptablesSaveCommand(runner.protocol)
 	args := []string{"-t", string(table)}
-	klog.V(4).Infof("running %s %v", iptablesSaveCmd, args)
+	klog.V(4).InfoS("Running", "command", iptablesSaveCmd, "arguments", args)
```
Comment (serathius) [inline pkg/util/iptables/iptables.go:358]:
```suggestion
	klog.V(4).InfoS("Running command", "command", iptablesSaveCmd, "arguments", args)
```

File: pkg/util/iptables/iptables.go [hunk 9]
```diff
@@ -412,15 +412,15 @@ func (runner *runner) restoreInternal(args []string, data []byte, flush FlushFla
...
+				klog.ErrorS(err, "Failed to close iptables locks")
 			}
 		}(locker)
 	}
 
 	// run the command and return the output or an error including the output and error
 	fullArgs := append(runner.restoreWaitFlag, args...)
 	iptablesRestoreCmd := iptablesRestoreCommand(runner.protocol)
-	klog.V(4).Infof("running %s %v", iptablesRestoreCmd, fullArgs)
+	klog.V(4).InfoS("Running", "command", iptablesRestoreCmd, "arguments", fullArgs)
```
Comment (serathius) [inline pkg/util/iptables/iptables.go:423]:
```suggestion
	klog.V(4).InfoS("Running command", "command", iptablesRestoreCmd, "arguments", fullArgs)
```

**Comment by yangjunmyfm192085** [comment]
@serathius   thanks for review.
/assign @danwinship, Could you help approve this pr?

**Review by yangjunmyfm192085 — COMMENTED** [review]
/remove-lifecycle stale

**Review by yangjunmyfm192085 — COMMENTED** [review]
Hi, @danwinship, as you said, the file `pkg/util/netsh/netsh.go` has been removed.
Continue the rest of this pr and merge it or abandon it?
/cc @danwinship

**Pull request merged**